Managers/RecentsManager.cs

```diff
@@ -70,7 +70,7 @@ public static void Initialize()
         public static void SaveRecents()
         {
             //set most recent to the last file open in the selected tab so the index is correct
-            Properties.Settings.Default.recents_0 = TabManager.ActiveTranslationManager.SourceFilePath;
+            Properties.Settings.Default.recents_0 = TabManager.ActiveTranslationManager?.SourceFilePath ?? "";
```

Comment:
Are you sure that should be a simple nullify check instead of checking for bigger error if this is ever null here?